### DISCUSSION TITLE
Аналоги флагов для zapret

### DISCUSSION THREAD

**Discussion by electrifying** [bol-van/zapret#277] [Q&A]
в goodbyedpi есть флаги -e1 set HTTPS fragmentation to value и  -f1 set HTTP fragmentation to value 
Хотелось бы узнать у автора запрета, а какие флаги тут эквиваленты этим?

**Comment by bol-van** [comment]
позиция фрагментации задается через --dpi-desync-split-pos
но сама по себе опция не делает ничего
необходимо задать  десинхронизацию со сплитом
простая фрагментация  --dpi-desync=split2
реверсная disorder2

http/https выбирается на этапе перенаправления трафика
по номеру порта

**Reply by electrifying** [reply]
> реверсная disorder2

Cпасибо, получается dpi-desync-split-pos работает и с disorder не только со split?

**Reply by bol-van** [reply]
верно
